### PR TITLE
Updated author site link to latest mybb website

### PR DESCRIPTION
**Pull request by MyBB-Andro**
Updated author site link to latest mybb website eeplacing old mybbboard link.

### CODE REVIEW

**Comment by Cu8eeeR** [comment]
@MyBB-Andro - Thank you. Unfortunately there are more links which needs to be replaced, for example here: https://github.com/Cu8eeeR/MyBB_Thank-you-like-plugin/blob/master/inc/plugins/thankyoulike.php#L62

**Comment by MyBB-Andro** [comment]
Done.

File: inc/plugins/thankyoulike.php [hunk 1]
```diff
@@ -59,17 +59,17 @@ function thankyoulike_info()
 	$prefix = 'g33k_'.$codename.'_';
 
 $url_AT= '<a href="http://my-bb.ir" target="_blank">AliReza_Tofighi</a>';
-$url_SP = '<a href="http://community.mybb.com/user-91011.html" target="_blank">SvePu</a>';
-$url_E = '<a href="http://community.mybb.com/user-84065.html" target="_blank">Eldenroot</a>';
-$url_DN = '<a href="http://community.mybb.com/user-51493.html" target="_blank">Dark Neo</a>';
+$url_SP = '<a href="https://community.mybb.com/user-91011.html" target="_blank">SvePu</a>';
+$url_E = '<a href="https://community.mybb.com/user-84065.html" target="_blank">Eldenroot</a>';
+$url_DN = '<a href="https://community.mybb.com/user-51493.html" target="_blank">Dark Neo</a>';
 $url_S = '<a href="https://github.com/Cu8eR/thankyou-like-plugin" target="_blank">GitHub</a>';
```
Comment (Cu8eeeR) [inline inc/plugins/thankyoulike.php]: https://github.com/Cu8eeeR/MyBB_Thank-you-like-plugin

File: inc/plugins/thankyoulike.php [hunk 2]
```diff
@@ -69,7 +69,7 @@ function thankyoulike_info()
 		"description"	=> $db->escape_string($lang->tyl_info_desc) . $lang->sprintf($lang->tyl_info_desc_url,$url_AT,$url_SP,$url_E,$url_DN,$url_S),
 		"website"	=> "http://community.mybb.com/thread-169382.html",
```
Comment (Cu8eeeR) [inline inc/plugins/thankyoulike.php]: https

**Comment by Cu8eeeR** [comment]
@MyBB-Andro - last typo and it should be fine.

I will merge this after merging latest changes from @SvePu

**Comment by MyBB-Andro** [comment]
Done, thanks for pointing out!

**Comment by Cu8eeeR** [comment]
Thank you very much!

**Comment by Cu8eeeR** [comment]
Thank you, merged! ;)
#120

**Pull request merged**